lib/features/home/presentation/utils/order_dialog_helper.dart

```diff
@@ -58,7 +58,7 @@ abstract class OrderDialogHelper {
         message: message,
       ),
     ).then((result) {
-      final res = result as List<String>?;
+      final res = result as List<dynamic>?;
```

Comment:
It should be List<bool> instead.
maybe just returning a bool will be better too

Reply:
You are right.